README.md

```diff
@@ -22,13 +22,13 @@ e.g.
 Instead of typing
 
 ```
-./typo3/cli_dispatch.phpsh extbase extension:install realurl
+typo3/cli_dispatch.phpsh extbase extension:install realurl
 ```
 
 just type:
 
 ```
-./typo3cms extension:activate realurl
+vendor/bin/typo3cms extension:activate realurl
```

Comment:
that is only true for composer installations and still might be different for other users that changed the bin path.

I still have no idea how to reflect this. But if we change something, we need to change it throughout all documentation (e.g. reference), not just in the readme